content/blog/2021-01-21-unpack-cfr.Rmd

```diff
@@ -0,0 +1,249 @@
+---
+title: "Unpacking the Drop in COVID-19 Case Fatality Rates"
+author: "Helen Zhou, Cheng Cheng, Jeremy C. Weiss, Zachary C. Lipton"
+output:
+  bookdown::html_document2: default
+bibliography: refs.bib
+link-citations: yes
+---
+
+<!-- install.packages("bookdown") -->
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Introduction
+
+Since the COVID-19 pandemic first reached the United States, the case fatality (CFR) rate has fallen considerably. Between the first peak in mid-April and the second peak in mid-July, the case fatality rate fell from $7.9\%$ to the $0.7\%$--$2.3\%$ range, where it has since remained despite cases rising again into an (ongoing) third wave:
+
+![cfr from covidcast](img/img_country/national_cases.svg)
+**Figure 1.** _From left to right: confirmed cases, deaths, and case fatality rate, calculated using 7-day trailing averages based on national reporting data available via USAFacts (@usafacts) and pulled from the CMU COVIDcast API (@covidcast2020api). Data outside the April 1st to December 1st time range considered in this study is grayed out._
```

Comment:
I believe the blog style puts "cfr from covidcast" as the caption under the plot, so you can move the "Figure 1" text in there

Reply:
Got it, went ahead and made those changes.